### DISCUSSION TITLE
A couple quick macros that help speed things up,

### DISCUSSION THREAD

**Discussion by lystraeudaimon** [fvtt-fria-ligan/twilight2000-foundry-vtt#47] [Ideas]
I added these macros to my quickbar it brings up the damage tables quickly which really helps in game. To add to your macrobar, create a new macro and copy and paste in the code below, remember to change to chat to script at the top.

**Quick vehicle damage**
new Dialog({
    title: "Vehicle Damage",
    buttons: {
        Arm: {
            label: "Penetration",
            callback: () => game.tables.getName("Component Damage - Penetration").draw()
        },
        Torso: {
            label: "No Penetration",
           callback: () => game.tables.getName("Component Damage - No Penetration").draw()

**Quick Crits**
ew Dialog({
    title: "Crits",
    buttons: {
        Arm: {
            label: "Arms",
            callback: () => game.tables.getName("Critical Injuries - Arms").draw()
        },
        Torso: {
            label: "Torso",
           callback: () => game.tables.getName("Critical Injuries - Torso").draw()
        },
        Head: {
            label: "Head",
            callback: () => game.tables.getName("Critical Injuries - Head").draw()
        },
        Legs: {
            label: "Legs",
            callback: () => game.tables.getName("Critical Injuries - Legs").draw()
        }
    }
}).render(true);

**Comment by tinwe** [comment]
Great macros!
Is there a simple way to make the rolls private initially? 
I am not familiar with the script syntax in Foundry myself, allthough I see that the Quick Crits lost its initial "n" for "new".

**Reply by Stefouch** [reply]
I think it should take by default the chosen mode in the chat menu. I'll test it.
